statscache/schedule.py

```diff
@@ -37,6 +37,12 @@ class Hour(Denomination):
 
 class Schedule(object):
     def __init__(self, second=None, minute=None, hour=None):
+
+        # Let higher order denominations precede lesser ones
+        # https://github.com/fedora-infra/statscache/issues/10
+        second = second or (minute and [0])
+        minute = minute or (hour and [0])
```

Comment:
No idea what kind of situation as possible or not, but just thought of another case:
```
>>> None or ([] and [0])
[]
```